src/Task/PhpStan.php

```diff
@@ -56,7 +60,17 @@ public function canRunInContext(ContextInterface $context)
     public function run(ContextInterface $context)
     {
         $config = $this->getConfiguration();
-        $files = $context->getFiles()->extensions($config['triggered_by']);
+        $files = $context
+            ->getFiles()
+            ->notPaths($config['ignore_patterns'])
+            ->extensions($config['triggered_by']);
+
+        $forcedFiles = $context->getFiles()->paths($config['force_patterns']);
```

Comment:
Can we move these lines that are forcing the files to be available to the FilesCollection?
This way we can do something like:
`$files->ensureFiles($context->getFiles()->paths($config['force_patterns]))`